node/common/core/common.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
   ConnectionString: require('./lib/connection_string.js'),
   endpoint: require('./lib/endpoint.js'),
   errors: require('./lib/errors.js'),
+  success: require('./lib/success.js'),
```

Comment:
good point. it might end up being a really long file though. I have not found a proper convention to handle result objects